pkgs/tools/text/ocrmypdf/default.nix

```diff
@@ -51,12 +51,14 @@ in buildPythonApplication rec {
     img2pdf
     pdfminer
     pikepdf
+    pillow
     reportlab
     ruffus
+    setuptools
+    tqdm
   ];
 
   checkInputs = with python3Packages; [
-    hocr-tools
```

Comment:
According to the [docs](https://github.com/jbarlow83/OCRmyPDF/blob/eb104b405d7d914b2d007b2a1c51241b22dabfc7/docs/advanced.rst#changing-the-pdf-renderer) it always defaults to `sandwich` so this is fine 👍️